.github/CODEOWNERS

```
@@ -153,6 +83,74 @@ docker-compose.serverless.yml                                      @DataDog/trac
+/tracer/test/test-applications/debugger/   @DataDog/debugger-dotnet
+debugger_*.cpp                             @DataDog/debugger-dotnet
+debugger_*.h                               @DataDog/debugger-dotnet
+/tracer/test/Datadog.Trace.Tests/Debugger  @DataDog/debugger-dotnet
+/tracer/test/Datadog.Trace.Debugger.IntegrationTests/ @DataDog/debugger-dotnet
+tracer/src/Datadog.InstrumentedAssemblyGenerator/ @DataDog/debugger-dotnet
+tracer/src/Datadog.InstrumentedAssemblyVerification/ @DataDog/debugger-dotnet
+fault_tolerant_*.cpp                       @DataDog/debugger-dotnet
+fault_tolerant_*.h                         @DataDog/debugger-dotnet
+/tracer/src/Datadog.Trace/FaultTolerant/   @DataDog/debugger-dotnet
+/tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs   @DataDog/debugger-dotnet
+Datadog.Trace.Debugger.slnf                @DataDog/debugger-dotnet
+
+# Serverless
+/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/    @DataDog/tracing-dotnet  @DataDog/serverless-aws
+/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SDK/    @DataDog/tracing-dotnet  @DataDog/serverless-aws
+/tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/   @DataDog/tracing-dotnet  @DataDog/serverless-aws
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/       @DataDog/tracing-dotnet  @DataDog/serverless-aws
+/tracer/test/test-applications/azure-functions/                    @DataDog/tracing-dotnet  @DataDog/serverless-aws
+/tracer/test/test-applications/integrations/Samples.AWS.Lambda/    @DataDog/tracing-dotnet  @DataDog/serverless-aws
+/tracer/test/test-applications/integrations/Samples.Amazon.Lambda.RuntimeSupport/        @DataDog/tracing-dotnet  @DataDog/serverless-aws
+docker-compose.serverless.yml                                      @DataDog/tracing-dotnet  @DataDog/serverless-aws
+
+# Shared code we could move to the root folder
+/tracer/build/                            @DataDog/apm-dotnet
+
+# CI
+/tracer/src/Datadog.Trace/Ci/             @DataDog/ci-app-libraries-dotnet @DataDog/apm-dotnet
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/   @DataDog/ci-app-libraries-dotnet @DataDog/apm-dotnet
+/tracer/src/Datadog.Trace/PDBs/MethodSymbolResolver.cs        @DataDog/ci-app-libraries-dotnet
+/tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/  @DataDog/ci-app-libraries-dotnet
+/tracer/test/test-applications/integrations/Samples.XUnitTests/  @DataDog/ci-app-libraries-dotnet
+/tracer/test/test-applications/integrations/Samples.NUnitTests/  @DataDog/ci-app-libraries-dotnet
+/tracer/test/test-applications/integrations/Samples.MSTestTests/  @DataDog/ci-app-libraries-dotnet
+/tracer/test/test-applications/integrations/Samples.MSTestTests2/  @DataDog/ci-app-libraries-dotnet
+/tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/SourceCodeIntegrationGitMetadataTests.cs  @DataDog/ci-app-libraries-dotnet
+
```

We should do the same for the snapshots:

```suggestion
/tracer/test/snapshots/MsTestV2* @DataDog/ci-app-libraries-dotnet
/tracer/test/snapshots/NUnit* @DataDog/ci-app-libraries-dotnet
/tracer/test/snapshots/XUnit* @DataDog/ci-app-libraries-dotnet
/tracer/test/snapshots/Selenium* @DataDog/ci-app-libraries-dotnet
```